core/server/apps/private-blogging/lib/views/private.hbs

```diff
@@ -15,7 +15,7 @@
         <link rel="shortcut icon" href="{{asset "favicon.ico"}}">
         <meta http-equiv="cleartype" content="on">
 
-        <link rel="stylesheet" href="{{asset "ghost.css" ghost="true" minifyInProduction="true"}}" />
```

Comment:
You're right! Does not work with subdirectory setup. Good catch! I will add the `{{asset}}` helper back.